### ISSUE TITLE
[Feature request] Block, mute, and hide

### ISSUE THREAD

**Issue by john-light** [stackernews/stacker.news#256]
As a user I would like to be able to curate my experience and my threads on stacker news by having the ability to block or mute other users and hide comments I consider low-quality.

Block: a user who is blocked will not be able to see or reply to my submitted topics or my comments on other topics. I will also not see submitted topics or comments from a blocked user.

Mute: I will not see submitted topics or comments from a muted user.

Hide: I can hide comments on my submitted topics so that neither I nor other users who visit the topic see them in the "Hot", "Recent" or "Top" comment views. Hidden comments can be seen by clicking a "Hidden" button in the comment filter menu.

**Comment by huumn** [comment]
I agree. We'll be making it so stackers get their own web of trust soon and I suspect it'll be relatively trivial to introduce pov features like this then.

**Comment by huumn** [comment]
- [ ] mute
- [ ] block
- [ ] hide

Mute is done. I'm hoping that provides some relief and I'm hoping we can address the problems underlying the other requests with personalized ranking. Going to leave open pending attempts at that.

**Comment by Soxasora** [comment]
> Block: a user who is blocked will not be able to see or reply to my submitted topics or my comments on other topics. I will also not see submitted topics or comments from a blocked user.

I'd like to work on this if I may ^^

**Comment by huumn** [comment]
We do eventually want this, but priority is on bug fixes and performance right now. We've put off some of this stuff as we were working on wallets and we need to focus on it.

Are there any bug fixes you're interested in? I'd be happy to increase difficulty or priority on any of them.

I'll go through and reprioritize things today and add some bounties for performance things.

**Comment by energypantry** [comment]
Hi! I can take this on with a fast turnaround. Do you accept USDC/USDT (any EVM chain) for payment? If yes, I can start immediately.
